reemgawad/brunchtime
# Add more users

What has changed:
Added more users to seeds
Added more ratings to all restaurants


How to test:
See all the ratings on restaurant page
make sure average rating compiles properly
